server.js

```diff
@@ -108,11 +122,15 @@ app.post('/', function(req, res) {
     }, function(err, configRes) {
       // default config
       var repoConfig = {
-        userBlacklist: []
+        userBlacklist: [],
+        maxReviewers: 3
       };
 
       if (!err && configRes) {
-        try { repoConfig = JSON.parse(configRes); } catch (e) {}
+        try {
+          merge(repoConfig, JSON.parse(configRes));
```

Comment:
I wasn't aware of this, thanks for the heads up. Would it be:

```js
repoConfig = { ...repoConfig, ...JSON.parse(configRes) }
```

or am I reading the docs wrong?